Sources/EdgeNetworkHandlers/EdgeNetworkService.swift

```diff
@@ -195,6 +167,59 @@ class EdgeNetworkService {
         }
     }
 
+    /// Handles the network response based on the response code
+    /// - Parameters:
+    ///   - responseCode: response code from the `HttpConnection`
+    ///   - connection: `HttpConnection` containing the network response info
+    ///   - streaming: `Streaming` settings if they were enabled for this response
+    ///   - responseCallback: `ResponseCallback` to be invoked once the server response is received
+    ///   - completion: a closure that is invoked with true if the hit should not be retried, false if the hit should be retried, along with the time interval that should elapse b
+    private func handleResponseWith(responseCode: Int,
+                                    connection: HttpConnection,
+                                    streaming: Streaming?,
+                                    responseCallback: ResponseCallback,
+                                    completion: @escaping (Bool, TimeInterval?) -> Void) {
+
+        switch responseCode {
+        case HttpResponseCodes.ok.rawValue:
+            Log.debug(label: self.LOG_TAG, "doRequest - Interact connection to Experience Edge was successful.")
+            self.handleContent(connection: connection,
+                               streaming: streaming,
+                               responseCallback: responseCallback)
+            responseCallback.onComplete()
+            completion(true, nil) // successful request, return true
+        case HttpResponseCodes.noContent.rawValue:
+            // Successful collect requests do not return content
+            Log.debug(label: self.LOG_TAG, "doRequest - Collect connection to Experience Edge was successful.")
```

Comment:
Do you want to use the common EdgeConstants.LOG_TAG here? Same question for all logging in this file.

Reply:
👍  Updated the logs for this file as well